### PR TITLE
Update coding page for Wave 5 deployment

### PR DESCRIPTION
**Pull request by capnrefsmmat**
Including the updated Word and PDF and date of deployment.

### CODE REVIEW

File: docs/symptom-survey/coding.md [hunk 1]
```diff
@@ -305,6 +310,14 @@ survey, even if they have already taken it before:
 > have participated before. Completing the survey again will help us understand
 > how the situation is changing.
 
+It also more clearly states that the data may be shared and aggregates publicly
+released:
```
Comment (Akvannortwick) [inline docs/symptom-survey/coding.md:314]:
```suggestion
Wave 5 now clearly states that the data may be shared and aggregates publicly when
released:
```

I'm not entirely sure what you wanted to say here, but the sentence felt choppy.

File: docs/symptom-survey/coding.md [hunk 2]
```diff
@@ -305,6 +310,14 @@ survey, even if they have already taken it before:
...
+
+> The de-identified results of this survey may be used for our future studies or
+> shared with other investigators for their research studies. Results published
```
Comment (Akvannortwick) [inline docs/symptom-survey/coding.md:317]:
```suggestion
> shared with other investigators for their research. Results published
```
Repeating studies felt redundant.

File: docs/symptom-survey/coding.md [hunk 3]
```diff
@@ -305,6 +310,14 @@ survey, even if they have already taken it before:
...
+> by us and other researchers will be in aggregate and will not identify
```
Comment (Akvannortwick) [inline docs/symptom-survey/coding.md:318]:
```suggestion
> by us and other researchers will be aggregated and will not identify
```

File: docs/symptom-survey/coding.md (hunk 2 above)
Comment (capnrefsmmat) [reply docs/symptom-survey/coding.md:317]: This text is part of the IRB-approved consent text, so while I agree with your suggestions, I don't think I can change it -- this is what's in Wave 5 and already went out

**Pull request merged**